.github/workflows/presubmit.yaml

```diff
@@ -112,17 +112,6 @@ jobs:
       - name: "Run pause tests"
         run: |
           ./scripts/github-actions/ga-pause-test.sh
-  direct-tests:
-    runs-on: ubuntu-22.04
-    timeout-minutes: 60
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v4
-        with:
-          go-version-file: 'go.mod'
-      - name: "run tests-e2e-direct"
-        run: |
-          ./scripts/github-actions/tests-e2e-direct.sh
```

Comment:
I think in the end state we don't have `direct` tests specifically but I think these can be useful for folks, especially as they iterate on a resource conversion.

Reply:
The KCC_USE_DIRECT_RECONCILERS flag shall still work (which gives developers the testing approach during the resource conversion). For release, we can purely rely on the CRD annotations to tell whether a direct or TF controller shall manage a resource kind (and eventually direct will be the default so it does not need a annotation).